plaidplotter.py

```diff
@@ -0,0 +1,318 @@
+import os
+import sys
+import errno
+import math
+import random
+import argparse
+import json
+import platform
+import plaidml
+import pandas as pd
+import seaborn as sns
+import numpy as np
+import matplotlib.pyplot as plt 
+import matplotlib as mpl
+from plaidml import plaidml_setup
+
+class plotter(object):
+    def getColor(self, hue, satur, val):
```

Comment:
Did a quick search and found that python already supports this, might want to use the library function instead:

>>> import colorsys
>>> colorsys.hsv_to_rgb(.5, .5, .2)
(0.1, 0.2, 0.2)